### PR TITLE
fixed - #278 suitecrm - studio: file name length.

### PR DESCRIPTION
**Pull request by shogunpol**
suitecrm version 7.2.2 - studio: file name length
## Description

suitecrm version 7.2.2 - studio: file name length.
References issue #278 

In studio => module => create field:
if the fieldname is too long, its cut and no user message is displayed.

It would be nice if the field name length could be longer and if its too long that a user message is displayed.

## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

### Final checklist

- [x] My code follows the code style of this project found [here](https://suitecrm.com/wiki/index.php/Coding_Standards).
- [ ] My change requires a change to the documentation.
- [x] I have read the [**How to Contribute**](https://suitecrm.com/wiki/index.php/Contributing_to_SuiteCRM) guidelines.

### CODE REVIEW

File: modules/DynamicFields/language/en_us.lang.php [hunk 1]
```diff
@@ -108,6 +108,7 @@
     'LBL_AUDITED' => 'Audited',
     'LBL_REPORTABLE' => 'Reportable',
     'ERR_RESERVED_FIELD_NAME' => "Reserved Keyword",
+    'ERR_LENGHT' => "Reserved Keywordaaaaaaaaaaa",
```
Comment (samus-aran) [inline modules/DynamicFields/language/en_us.lang.php]: Please correct spelling and the translatable word is not correct.

File: modules/DynamicFields/templates/Fields/Forms/coreTop.tpl [hunk 2]
```diff
@@ -44,54 +44,63 @@
...
-</tr>
-<tr>
-    <td class='mbLBL'>{sugar_translate module="DynamicFields" label="COLUMN_TITLE_COMMENT_TEXT"}:</td><td>{if $hideLevel < 5 }<input type="text" name="comments" value="{$vardef.comments}">{else}<input type="hidden" name="comment" value="{$vardef.comment}">{$vardef.comment}{/if}
-    </td>
-</tr>
+                {$vardef.name}
+            {/if}
+            <script>
+                addToValidate('popup_form', 'name', 'DBName', true, '{sugar_translate module="DynamicFields" label="COLUMN_TITLE_NAME"} [a-zA-Z_]');
+                addToValidateLength('popup_form', 'name', 'length', true, ' {sugar_translate module="DynamicFields" label="ERR_LENGHT"}', 25);
```
Comment (samus-aran) [inline modules/DynamicFields/templates/Fields/Forms/coreTop.tpl]: Please correct spelling of the label

**Comment by samus-aran** [comment]
I have begun testing this but I am unable to retrieve the max length error.  The input field itself only allows 25 characters max so why require an validate message?  Can you demonstrate this validation to myself to pass testing.

File: modules/DynamicFields/language/en_us.lang.php (hunk 1 above)
Comment (shogunpol) [inline modules/DynamicFields/language/en_us.lang.php]: Done

File: modules/DynamicFields/templates/Fields/Forms/coreTop.tpl (hunk 2 above)
Comment (shogunpol) [inline modules/DynamicFields/templates/Fields/Forms/coreTop.tpl]: Done

**Comment by Dillon-Brown** [comment]
Closed due to inactivity.

**Pull request closed without merging**